src/planner.rs

```diff
@@ -121,7 +121,9 @@ impl<C: 'static> Planner<C> {
     /// Creates a new planner with a thread pool that has
     /// `num_threads` threads.
     pub fn with_num_threads(world: World, num_threads: usize) -> Planner<C> {
-        let config = Configuration::new().set_num_threads(num_threads);
+        let config = Configuration::new()
+            .set_num_threads(num_threads)
+            .set_panic_handler(Arc::new(|e| println!("{:?}", e)));
```

Comment:
Perhaps we can make this message more readable?
Just notice, it's pretty good for now.